modules/reference/pages/topic-properties.adoc

```diff
@@ -242,6 +246,44 @@ When `segment.bytes` is set to a positive value, it overrides the cluster proper
 
 ---
 
+[[writecaching]]
+==== write.caching
+
+The write caching mode to apply to a topic. 
+
+When `write.caching` is set to on, it overrides the cluster property xref:cluster-properties.adoc#write_caching[`write_caching`]. This caches batches of writes in memory until the segment appender chunk is full, instead of flushing for every `acks=all` write. Flushes follow <<flushms, `flush.ms`>> and <<flushbytes, `flush.bytes`>>, whichever is reached first. 
+
+**Default**: `off`
+
+**Values**:
+
+- `on` - Turns on write caching for a topic, according to size-based and time-based retention flushing.
+- `off` - Turns off write caching for a topic, according to size-based and time-based retention flushing.
+
+**Related topics**:
+
+- xref:develop:produce-data/configure-producers.adoc#write-caching[Write caching]
+
+---
+
+[[flushms]]
+==== flush.ms
+
+For <<writecaching,`write.caching`>>, this is the maximum delay (in ms) between two subsequent flushes. After this delay, the log is automatically force flushed.
```

Comment:
flush.ms/bytes donot just apply for write caching, they also apply for acks=0/1 modes  (all modes that donot explicitly fsync).. I'd remove "For <<writecaching,`write.caching`>>"

same below.

Reply:
Cluster level default is `raft_replica_max_flush_delay_ms`, if no topic level override exists.